std/meta.d

```diff
@@ -487,7 +487,11 @@ template NoDuplicates(TList...)
     {
         enum steps = 16;
         alias first = NoDuplicates!(TList[0 .. steps]);
-        alias NoDuplicates = NoDuplicates!(EraseAllN!(first.length, first, TList[steps..$]));
+        alias NoDuplicates = NoDuplicates!(
```

Comment:
In case there are no duplicates, it looks like this would expand to the original arguments. The remaining work to perform is eliminating duplicates between the first and the second half below.